.buildkite/scripts/steps/code_coverage/reporting/uploadPrevSha.sh

```diff
@@ -9,4 +9,7 @@ collectPrevious() {
 }
 collectPrevious
 
-gsutil cp previous.txt gs://elastic-bekitzur-kibana-coverage-live/previous_pointer/
\ No newline at end of file
+# TODO: Safe to remove this after 2024-03-01
+gsutil cp previous.txt gs://elastic-kibana-coverage-live/previous_pointer/
+
+gsutil cp previous.txt gs://elastic-kibana-coverage-live/previous_pointer/
```

Comment:
Duplicate line?

Reply:
good catch! I meant to have the legacy address in the one above

Reply:
done!